spec/fixtures/controllers/applications/lease_up_application_domain.json

```diff
@@ -1,7 +1,92 @@
 {
-  "total_monthly_rent": null,
-  "sub_status": "Written withdrawal",
-  "status": "Submitted",
+  "alternate_contact": null,
```

Comment:
so some of the data changed when the vcr was updated, and the new data was all in a different order.

I sorted this all by alphabetical order so that next time the data is updated hopefully the diff won't be so bad as long as we keep it sorted

Reply:
oh interesting! I thought we hadn't gotten fixtures hooked up to vcr cassettes for this repo 🤔

Reply:
@akegan well the vcr cassettes get data from real responses, these fixtures are just what we're comparing them with, so when the response changed, I had to change the expected result fixtures as well

Reply:
ohh I see. I thought you were saying that this automatically updated, which is something that we've wanted to setup for a while 😅